[ADD] mail_notification_custom_subject: Apply a custom subject to mail notifications (#650)
OCA/social: mail_notification_custom_subject/README.rst

    :alt: Production/Stable
.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsocial-lightgray.png?logo=github
    :target: https://github.com/OCA/social/tree/12.0/mail_notification_custom_subject
    :alt: OCA/social
.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
    :target: https://translation.odoo-community.org/projects/social-12-0/social-12-0-mail_notification_custom_subject
    :alt: Translate me on Weblate
.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
    :target: https://runbot.odoo-community.org/runbot/205/12.0
    :alt: Try me on Runbot

|badge1| |badge2| |badge3| |badge4| |badge5| 

This module allows you to specify templates to override the subject on the notification
emails sent by Odoo

**Table of contents**

.. contents::
   :local:

Configuration
=============

* Activate access to **Technical Features** (debug mode).
* Go to **Settings > Technical > Email > Subject Replacement Templates**
* Create a new template.

>     * The field **Models** specifies the models to which the subject template should apply in the emails sent by Odoo

Reviewer: ```suggestion
    * The field **Model** specifies the model to which the subject template should apply in the notification emails sent by Odoo.
```

   !! changes will be overwritten.                   !!
   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

.. |badge1| image:: https://img.shields.io/badge/maturity-Production%2FStable-green.png
    :target: https://odoo-community.org/page/development-status
    :alt: Production/Stable
.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsocial-lightgray.png?logo=github
    :target: https://github.com/OCA/social/tree/12.0/mail_notification_custom_subject
    :alt: OCA/social
.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
    :target: https://translation.odoo-community.org/projects/social-12-0/social-12-0-mail_notification_custom_subject
    :alt: Translate me on Weblate
.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
    :target: https://runbot.odoo-community.org/runbot/205/12.0
    :alt: Try me on Runbot

|badge1| |badge2| |badge3| |badge4| |badge5| 

This module allows you to specify templates to override the emails sent by Odoo

> in the chatter.

Reviewer: ```suggestion
This module allows you to specify templates to override the subject on the notification emails
sent by Odoo.
```

**Table of contents**

.. contents::
   :local:

Usage
=====

To use this module, you need to:

* Activate access to **Technical Features** (debug mode).
* Go to **Settings > Technical > Email > Subject Replacement Templates**
* Create a new template.

    * The field **Subject Template** accepts `Jinjah<ttps://jinja.palletsprojects.com/en/2.11.x/>` expressions.

>     * The field **Replace** specifies if the template should replace existing templates or append to them.

Reviewer: ```suggestion
    * The field **Replace** specifies if the template should replace existing content or append to it.
```

>     * The field **Replace** specifies if the template should replace existing templates or append to them.

Reviewer: Missing model definition

Reviewer: Till here should go on CONFIGURE.rst, not USAGE.rst
